src/javascript/app/pages/trade/digit_ticker.js

```diff
@@ -177,9 +177,9 @@ const DigitTicker = (() => {
     };
 
     const setElements = () => {
-        el_peek     = el_container.querySelector('.peek');
-        el_peek_box = el_container.querySelector('.peek-box');
-        el_mask     = el_peek_box.querySelector('.peek-box > .mask');
+        el_peek     = el_container ? el_container.querySelector('.peek') : null;
+        el_peek_box = el_peek ? el_container.querySelector('.peek-box') : null;
+        el_mask     = el_peek_box ? el_peek_box.querySelector('.peek-box > .mask') : null;
```

Comment:
This is to prevent trackjs errors

Reply:
shouldn't it be `el_peek_box.querySelector('> .mask')`? is there another peek box inside the peek box?